jwst/datamodels/integration.py

```diff
@@ -0,0 +1,27 @@
+import sys
```

Comment:
docstrings for module and function should be added.

Reply:
See 69ea324 for documentation.

Reply:
LGTM